viz.yaml

```diff
@@ -109,28 +109,28 @@ process:
     location: cache/process/lastUpdated.yaml
     scripts: scripts/process/lastUpdated.R
     depends:
-      data: aggregate_ga
+      data: fetchGA
     mimetype: text/yaml
     reader: md # need a better plain text reader
   -
     id: sessions_and_new_users
     processor: sessions_and_new_users 
     scripts: scripts/process/sessions_and_new_users.R
-    depends: aggregate_ga
+    depends: fetchGA
```

Comment:
I'm worried this will just move the memory problem to the fetch, but it seems to make sense to remove the aggregate if it isn't doing anything.  As for the depends, it is probably best to add a name in all the depends sections.